src/histolab/filters/image_filters.py

```diff
@@ -16,6 +16,12 @@
 # limitations under the License.
 # ------------------------------------------------------------------------
 
+"""All filters implemented in the image filters submodule take as input a Pillow Image
+object. Additionally, some of the image filters in histolab leverage functions and
+utilities by scikit-image. Image filters are divided into sub-categories, depending on
+their behaviour and output type.
```

Comment:
```suggestion
utilities by scikit-image.
```
here we are not talking about the categories... we could move part of the wall text here